flow/layers/clip_rrect_layer_unittests.cc

```diff
@@ -195,5 +195,65 @@ TEST_F(ClipRRectLayerTest, PartiallyContainedChild) {
            MockCanvas::DrawCall{1, MockCanvas::RestoreData{0}}}));
 }
 
+static bool ReadbackResult(PrerollContext* context,
```

Comment:
Extra extra nit: I felt that you can write template functions in a header file like `clip_layer_readback_unittests.h` to merge similar code in clip_rect, clip_rrect, and clip_path unit tests. This is of course not very important and it shouldn't block you from merging this PR. Feel free to come back to play with it later when you have time and mood :)

Reply:
On a related note, I pointed out elsewhere that we could probably have all of the ClipFooLayer implementations subclass off of a ClipLayerBase class and all they'd need to implement would be 2 or 3 very short helper methods.